.github/workflows/sphinx.yml

```diff
@@ -0,0 +1,120 @@
+# Inspired by https://github.com/rkdarst/sphinx-actions-test/blob/master/.github/workflows/sphinx-build.yml
+
+name: sphinx
+on: [push, pull_request]
```

Comment:
You may want to restrict this to run only on push and pull requests against master, and/or push to tags?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

Reply:
That is a good idea. I have stripped everything right back so it no longer supports developer branches. Release tags will have to be added later but this is a good "minimal solution."